### PR TITLE
Install LZO

### PR DESCRIPTION
**Pull request by findepi**

### CODE REVIEW

File: prestodev/hdp3.1-hive/files/root/setup.sh [hunk 1]
```diff
@@ -57,3 +57,6 @@ killall mysqld
 sleep 10s
 mkdir /var/log/mysql/
 chown -R mysql:mysql /var/log/mysql/
+
+# Additional libs
+cp -av /usr/hdp/current/hadoop-client/lib/native/Linux-amd64-64/* /usr/lib64/
```
Comment (kokosing) [inline prestodev/hdp3.1-hive/files/root/setup.sh]: can we add test for lzo?

File: prestodev/centos6-oj8/Dockerfile [hunk 2]
```diff
@@ -32,6 +32,7 @@ RUN \
     yum install -y \
         net-tools `# netstat` \
         sudo \
+        telnet `# helpful when troubleshooting product tests` \
```
Comment (kokosing) [inline prestodev/centos6-oj8/Dockerfile]: ❤️

File: prestodev/hdp3.1-hive/files/root/setup.sh (hunk 1 above)
Comment (findepi) [reply prestodev/hdp3.1-hive/files/root/setup.sh]: 😱  😱  😱 🙀

Comment (findepi) [reply prestodev/hdp3.1-hive/files/root/setup.sh]: We cannot, because tests are executed for all images, and I only added this to "latest" HDP images.

**Comment by findepi** [comment]
It's green (https://travis-ci.org/findepi/docker-images/builds/577803106)

**Pull request merged**